### ISSUE TITLE
State config file path in the documentation

### ISSUE THREAD

**Issue by michidk** [louketo/louketo-proxy#655]
What is the path for the config file, when not configuration using CLI args?

I can't find it in the docs.

**Comment by jangaraj** [comment]
There is no default config file location.

**Comment by michidk** [comment]
How can I then specify a path to load the config file?

**Comment by jangaraj** [comment]
https://github.com/louketo/louketo-proxy/blob/c50082152a87e4bee9d34ef7ab00bf4db7a9769f/cli.go#L53-L57

=> You can't specify just path - directory

It looks like you are expecting old school approach, where app has own config file somewhere (`/etc/app/...`) and it has hardcoded path to that config file. That is not a case here. You can use env variables (not for all config options) OR parameters OR config file OR their combination.

**Comment by michidk** [comment]
So if I understood that code correctly, I can specify one (or multiple) config-file with the `--config` flag?
I think that should be documented in the documentation rather just in code.

**Comment by jangaraj** [comment]
Yeah 🤞. Please check `-h` in the CLI to see all config options, supported env variables. That is IMHO the best doc.
```
$ docker run --rm -ti quay.io/louketo/louketo-proxy -h
NAME:
   louketo-proxy - is a proxy using the keycloak service for auth and authorization

USAGE:
   louketo-proxy [options]

VERSION:
   v2.3.0 (git+sha: aa9ae98-dirty, built: 15-06-2020)

AUTHOR:
   Louketo <louketo@googlegroups.com>

COMMANDS:
     help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --config value                            path the a configuration file [$PROXY_CONFIG_FILE]
...
```

**Comment by TcaManager** [comment]
> Yeah crossed_fingers. Please check `-h` in the CLI to see all config options, supported env variables. That is IMHO the best doc.
> 
> ```
> $ docker run --rm -ti quay.io/louketo/louketo-proxy -h
> NAME:
>    louketo-proxy - is a proxy using the keycloak service for auth and authorization
> 
> USAGE:
>    louketo-proxy [options]
> 
> VERSION:
>    v2.3.0 (git+sha: aa9ae98-dirty, built: 15-06-2020)
> 
> AUTHOR:
>    Louketo <louketo@googlegroups.com>
> 
> COMMANDS:
>      help, h  Shows a list of commands or help for one command
> 
> GLOBAL OPTIONS:
>    --config value                            path the a configuration file [$PROXY_CONFIG_FILE]
> ...
> ```

Hi, is there a way I use this in a docker-compose file please?
